lldb/tools/lldb-dap/JSONUtils.cpp

```diff
@@ -154,7 +154,7 @@ DecodeMemoryReference(llvm::StringRef memoryReference) {
 std::vector<std::string> GetStrings(const llvm::json::Object *obj,
                                     llvm::StringRef key) {
   std::vector<std::string> strs;
-  auto json_array = obj->getArray(key);
+  const auto *json_array = obj->getArray(key);
```

Comment:
ah, you are using clang-tidy? I always always get those warnings 😄

Reply:
Yea, trying to fix those as I work on refactoring `g_dap`.